src/Appwrite/Platform/Tasks/ScheduleExecutions.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Appwrite\Event\Func;
+use Cron\CronExpression;
+use Utopia\Database\Database;
+use Utopia\Pools\Group;
+
+class ScheduleExecutions extends ScheduleBase
+{
+    public const UPDATE_TIMER = 3; // seconds
+    public const ENQUEUE_TIMER = 4; // seconds
+
+    public static function getName(): string
+    {
+        return 'schedule-executions';
+    }
+
+    public static function getSupportedResource(): string
+    {
+        return 'execution';
+    }
+
+    protected function enqueueResources(Group $pools, Database $dbForConsole): void
+    {
+        foreach ($this->schedules as $schedule) {
+            if (!$schedule['active'] || CronExpression::isValidExpression($schedule['schedule'])) {
```

Comment:
If I understand it right, we dont care about schedule cron here. Also I think it would never be valid cron in this worker - its datetime

Reply:
Yes - removed.